rasa/utils/tensorflow/model_data.py

```diff
@@ -45,40 +61,61 @@ def __init__(
             data: the data holding the features
         """
 
-        self.data = data or {}
+        self.data = data or defaultdict(lambda: defaultdict(list))
         self.label_key = label_key
+        self.label_sub_key = label_sub_key
         # should be updated when features are added
         self.num_examples = self.number_of_examples()
 
-    def get_only(self, key: Text) -> Optional[np.ndarray]:
-        if key in self.data:
-            return self.data[key][0]
+    def get_only(self, key: Text, sub_key: Text) -> Optional[np.ndarray]:
```

Comment:
does it get array only if it's singular list? let's add the check then

Reply:
this is given from the data structure of `self.data`, which is `Dict[Text, Dict[Text, List[np.ndarray]]]`. So the first example of the list is always a `np.ndarray`

Reply:
but why do we need the first example?

Reply:
I meant if it meant to used if `Len(List[np.ndarray]) == 1` I'd add a check here, so that it is not used to get a first example of a long list

Reply:
it seems to be not used, I guess we can remove the method